index.html

```diff
@@ -0,0 +1,108 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML course!</title>
+
+    <link rel="stylesheet" href="Styles.css">
+    <link rel="stylesheet" href="Stylesdesktop.css">
+
+                
+
+</head>
+<body>
+    <section class="header">
+        <div>
+            <ul class="pre-nav-bar desktop">
+                <li class="socialMedia"><a class="icon" href="https://github.com/Kakalanp"><img src="Images/GitHub.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://www.linkedin.com/in/joaquín-garrido-lecca-zanetti-623583204/"><img src="Images/LinkedIn.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://twitter.com/LeccaJoaquin"><img src="Images/twitter.png" alt=""></a></li>
+                <li><a class="link">English</a></li>
+                <li><a class="link">My Page</a></li>
+                <li><a class="link">Logout</a></li>
+            </ul>
+        </div>
+
+        <nav class="nav-bar">
+            <a href="index.html" class="logo desktop"><img class="logo" src="Images/Logo.png" alt="Logo"></a>
+            <ul class="menu">
+                <li><a href="About.html">About</a></li>
+                <li><a>Program</a></li>
+                <li><a>Join</a></li>
+                <li><a>Sponsors</a></li>
+                <li><a>News</a></li>
+                <li><a class="cc-campaign">CC Campaign</a></li>
+            </ul>
+            <a class="hamburger mobile" href="#">≡</a>
+        </nav>
+    </section>
+
+    <main>
+        <div class="main">
+            <h2>Hello future developer!</h2>
+            <h1 class="main-title">HTML Course is here!</h1>
+            <div class="spit-info"><p>Learn to program in HTML in only a month! With experts from around the world to guide you though the course!</p></div>
+            <div class="starting-date"><p>2022.03.01(Mon) ~ 2022.03.31(Thu)</p></div>
+        </div>
```

Comment:
- [x] Please try to stick the design as much as possible, you can give your project some padding to look more clear for the user

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/152261995-85c73958-d064-4a4b-8808-ca89a5a0f17d.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/152261999-c0ae65bf-5714-4c16-81b0-2409fb8ff969.PNG)

Reply:
I just added some padding, as you suggested https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/CSS/Styles.css#L134 What do you think of it now?